registration/registration_tests.py

```diff
@@ -1,41 +1,108 @@
+from copy import deepcopy
 from django.contrib.auth import get_user_model
 from urllib.error import HTTPError
 import pytest
 
 
 User = get_user_model()
 REGISTER_URL = 'http://localhost/register/'
+USER: dict = {
+    "email": "user@example.com",
+    "first_name": "john",
+    "last_name": "doe",
+    "gender": "m",
+    "phone_number": None,
+    "mobile_number": None,
+    "birthday": "05/09/1989",
+    "street": "secondary street",
+    "house_number": "1",
+    "zipcode": "12345",
+    "city": "example-city"
+}
+
+SUCCESS_STRING = f'created user {USER["email"]}'
 
 
 def test_registration__1(browser):
     """It is possible to register with an unused email address."""
-    email = 'test@example.com'
-
-    browser.open(REGISTER_URL)
-    browser.register_user(email)
-    assert f'created user {email}' in browser.contents
+    register_user(browser, USER)
+    assert SUCCESS_STRING in browser.contents
     assert browser.url == REGISTER_URL
 
 
 def test_registration__2(browser):
     """It is not possible to register with an used email address."""
-    email = 'test@example.com'
-    User(email=email).save()
+    user = deepcopy(USER)
```

Comment:
I wondered, why my impulse was to make a pytest-fixture out of this dictionary the first time I saw that. Turns out it changes during the test. As a fixture, this would be accounted for. It is okay for know, but the next time we should use a fixture and simply return a `dict`.